### ISSUE TITLE
volumeIntent: Tap {undefined} 

### ISSUE THREAD

**Issue by dboyd13** [patfreeman/alexa-app-kegbot#10]
Not sure if I've missed on of the configuration steps? - but see below.

Thanks

Utterance: `"how much beer is left?"`

--Snip from node server.js output--
```
REQUEST: volume Intent
RESPONSE: Kegbot has 9.0 litres of Boy of Summer on tap number undefined
RESPONSE: Kegbot has 9.5 litres of Alt MACC on tap number undefined
RESPONSE: Kegbot on tap number 1.  and  on tap number 2. 
```

**Comment by patfreeman** [comment]
Resolved with pull #11

**Comment by dboyd13** [comment]
Works great! Thanks. 

Sorry for the fumble with the pull request that came in the wrong direction (again)
